src/Route/RouteDefinition.php

```diff
@@ -7,6 +7,7 @@ final class RouteDefinition
     public const SERVICE = 'service';
     public const MIDDLEWARES = 'middlewares';
     public const MIDDLEWARE_GROUPS = 'middlewareGroups';
+    public const IGNORE_VERSION_MIDDLEWARE_GROUP = 'ignoreVersionMiddlewareGroup';
```

Comment:
[note] Za mě je zvláštní tomu říkat "version" (např. "api" není verze), ale je to konzistentní s aktuálním názvoslovím